lib/tuya.js

```diff
@@ -1250,6 +1245,7 @@ const valueConverter = {
     onOff: valueConverterBasic.lookup({'ON': true, 'OFF': false}),
     powerOnBehavior: valueConverterBasic.lookup({'off': 0, 'on': 1, 'previous': 2}),
     lightType: valueConverterBasic.lookup({'led': 0, 'incandescent': 1, 'halogen': 2}),
+    lightTypeToEnum: valueConverterBasic.lookup({'led': new Enum(0), 'incandescent': new Enum(1), 'halogen': new Enum(2)}),
```

Comment:
The line above is possibly an existing bug, unless other tuya devices expect the value to be numeric.
In case of MS-105B, the `light_type` should be sent as enum.

Reply:
I think others reported it works fine, so let's leave it as is.